src/resources/postcss/app-shop.pcss

```
@@ -554,18 +552,94 @@ body.tribe_events_page_tribe-app-shop {
 	display: none;
 }
 
-/**
- * Button compatibility for App Shop.
+// Stellar Discounts Tab
 
-.branch-4-9 {
-	#tribe-app-shop {
-		.addon-grid {
-			.button {
-				height: 58px;
-				box-shadow: none;
-			}
+#tribe-stellar .content-wrapper {
+	max-width: 1020px;
+}
+
+.stellar-logo {
+	margin: 0 auto 32px auto;
```

```suggestion
	margin: 0 auto 32px;
```
Don't need the second `auto`. Implied auto for margin-left and margin-right thanks to the first one.